spec/PostgresStorageAdapter.spec.js

```diff
@@ -570,6 +570,35 @@ describe_only_db('postgres')('PostgresStorageAdapter', () => {
     await adapter.deleteIdempotencyFunction();
     await client.none(qs);
   });
+
+  it('saves object with a pointer field set to undefined', async () => {
```

Comment:
This test should be moved to where the existing pointer tests are, as it's not a postreg specific test, even if the issue occurs only with postgres.